modules/manager/logging/mode.go

```diff
@@ -0,0 +1,29 @@
+package logging
+
+import "fmt"
+
+// LoggingMode is the type for the logging mode.
+type LoggingMode string
```

Comment:
Nitpick: perhaps we could call this `Mode` given that it's located in the `logging` package. This will prevent the stuttering use sites like: `logging.LoggingMode`

WDYT?

Reply:
PTAL: https://github.com/Kong/gateway-operator/pull/1435/commits/70046304700aae6bb81b669a2d87c147e0faa080